src/test/deploy/functions/params.spec.ts

```diff
@@ -0,0 +1,134 @@
+import * as chai from "chai";
+import * as sinon from "sinon";
+chai.use(require("chai-as-promised"));
+const expect = chai.expect;
```

Comment:
Do we still need to use `chai.use(require("chai-as-promised"));`? I think that's old. Most tests just have

```
import { expect } from "chai"`
```

Reply:
Oh. Well. That makes more sense.